.github/workflows/bump-version.yml

```diff
@@ -0,0 +1,30 @@
+on:
+  pull_request:
+    branches:
+      - main
+    types: [closed]
+jobs:
+  bump-version:
+    if: github.event.pull_request.merged == true
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          token: ${{ secrets.GH_PUSH_TOKEN }}
+      - name: configure git
+        run: |
+          git remote set-url origin https://itsalaidbacklife@github.com/cuttle-cards/cuttle
+          git config --global user.name "Cuttle Continuous Integration"
+          git config --global user.email "itsalaidbacklife@gmail.com"
+      - name: bump major version
+        if: contains(github.event.pull_request.labels.*.name, 'version-major')
+        run: npm run version:major
+      - name: bump minor version
+        if: contains(github.event.pull_request.labels.*.name, 'version-minor')
+        run: npm run version:minor
+      - name: bump patch version
+        if: contains(github.event.pull_request.labels.*.name, 'version-patch')
+        run: npm run version:patch
+      - name: push
+        run: git push origin main
```

Comment:
Notably these don't check the other labels. Each step independently runs and bumps the version if it finds a corresponding label, so if we apply major + minor + patch version labels, it will bump the package by all three. That could allow mistakes on our part, but it's simpler this way to start and we can see how it goes from here

Reply:
You know me, I don't agree with that since you'd never be doing all three on a given PR. We also have that status check for labels in place that will verify we only have one so this is probably fine.